examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -53,70 +72,84 @@ my_module/__init__.py        1      0   100%
 --------------------------------------------
 TOTAL                        5      0   100%
-
-
 ```
 
-Now that our tests are working locally, we want to configure GCP to run them every time we push new code.
+Now that your tests are working locally, you can configure Cloud Builder to run them every time you push new code.
+
+## 3. Building a Python Cloud Build Container
+To run Python based tests in cloud builder you need a Python [cloud builder](https://cloud.google.com/cloud-build/docs/cloud-builders). 
+A cloud builder is just a docker container image with the software you need in it. Google doesn't distribute a 
+Python cloud build container, so you have to build one yourself. The code needed to build a custom Python 3 cloud build 
+container is located in '/python-cloud-builder' under the root of this project. 
 
-## 3. Building a Python Cloud Builder
-In order to execute python based tests in cloud builder you need a python cloud builder. 
-A cloud builder is just a docker container with the stuff you need in it. Google doesn't distribute a 
-python cloud builder, so we have to build one ourselves. I've created one in '/python-cloud-builder.' All you need to do 
-is run this:
+Run these commands from the root of the project to build and upload your custom Python cloud build container:
 
 ```
 cd python-cloud-builder
 gcloud builds submit --config=cloudbuild.yaml .
-
 ```
 
-This creates a custom cloud builder in your container registry called gcr.io/$PROJECT_ID/python-cloudbuild that you can
-use to run tests with.
+This creates the custom Python cloud builder and uploads it to your GCP project's container registry, which is a private
+location to store container images. Your new cloud builder will be called gcr.io/$PROJECT_ID/python-cloudbuild, where 
+$PROJECT_ID is the name of your GCP project.
 
 ## 4. Create a Cloud Build Trigger
-* On the GCP console navigate to 'Cloud Build' > 'Triggers'
-* Add a trigger
-* Choose cloud source repository
+Now that you've created a Python cloud builder to run your tests, you 
+should [create a trigger](https://cloud.google.com/cloud-build/docs/running-builds/automate-builds) that tells Cloud
+Build when to run those tests. To do that, follow these steps:
+
+* On the GCP console navigate to 'Cloud Build' > 'Triggers.'
+* Add a trigger.
+* Choose cloud source repository.
 * Pick the repo you created in step 1.
-* Assuming you want the trigger to fire on any branch, accept the default trigger type and regex
-* Choose 'cloud build configuration file' from 'build configuration'
-* Click create trigger
+* Assuming you want the trigger to fire on any branch, accept the default trigger type and regex.
+* Choose 'cloud build configuration file' from 'build configuration.'
+* Click create trigger.
 
 
 ## 5. Create a cloudbuild.yaml file that executes your tests and runs pylint
+At this point you've created a Python cloud build container in step 3, a build trigger that runs whenever you 
```

Comment:
Again with cloud build vs cloud builder, and again I regret some of the changes I asked for. You could also use an "and" after the comma.

Reply:
yeah, I called it a cloud builder now.